shell/platform/darwin/ios/framework/Source/FlutterEngine.mm

```diff
@@ -824,7 +824,7 @@ - (FlutterBinaryMessengerConnection)setMessageHandlerOnChannel:(NSString*)channe
     return _connections->AquireConnection(channel.UTF8String);
   } else {
     NSAssert(!handler, @"Setting a message handler before the FlutterEngine has been run.");
-    // Setting a handler to nil for a not setup channel is a noop.
+    // Setting a handler to nil for a not yet setup channel is a noop.
```

Comment:
This one definitely needs to be set up. I think in fact not-yet-set-up.

Reply:
Oh god I fixed the yet but apparently didn't save after hitting space.

Agreed. It should be not-yet-set-up though we should probably just rephrase. I realise that I, in particular, have a really bad habit of injecting incredibly-long-yet-dubious chains of words into my writing.

Reply:
Done.